src/app/clock-in-out-history/clock-in-out-history.component.css

```diff
@@ -0,0 +1,30 @@
+.flex-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(25rem, 1fr));
+  grid-auto-rows: 50rem;
+  grid-gap: 1rem;
+  margin: 1rem;
+  font-family: Lato;
+}
+.history-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  background-color: #dedede;
+  border-radius: 100rem;
+}
+li {
+  background-color: #b3b1af;
+  font-size: 1.5rem;
+  margin: 0.9rem;
+  list-style: none;
+}
+.odd-li {
+  background-color: aliceblue;
+}
+ul {
+  width: 70%;
+  overflow: hidden;
+  overflow-y: scroll;
```

Comment:
Hm, co tu się wydarzyło? wydaje mi się że overflow: hidden usuwa nam scrolla a overflow-y: scroll dodaje nam go nawet, kiedy nie ma takiej potrzeby na osi Y. Czy ten zabieg miał na celu usunąć nam scrolla z osi X? Jeśli tak to powinno być tylko overflow-x: hidden